modules/local/resfinder/main.nf

```diff
@@ -0,0 +1,52 @@
+process RESFINDER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::resfinder=4.1.11"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/resfinder:4.1.11--hdfd78af_0':
+        'biocontainers/resfinder:4.1.11--hdfd78af_0' }"
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*_pheno_table_*.txt")                    , emit: specie_pheno_table, optional: true
+    tuple val(meta), path("*_pheno_table.txt")                      , emit: pheno_table
+    tuple val(meta), path("*_PointFinder_prediction.txt")           , emit: pointfinder_pred
+    tuple val(meta), path("*_PointFinder_results.txt")              , emit: pointfinder_results
+    tuple val(meta), path("*_ResFinder_Hit_in_genome_seq.fsa")      , emit: resfinder_hit
+    tuple val(meta), path("*_ResFinder_Resistance_gene_seq.fsa")    , emit: resfinder_resistance_gene_seq
+    tuple val(meta), path("*_ResFinder_results_table.txt")          , emit: resfinder_results_table
+    tuple val(meta), path("*_ResFinder_results.txt")                , emit: resfinder_results
+    tuple val(meta), path("*.json")                     , emit: json
+    path "versions.yml"                                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args     = task.ext.args ?: ''
+    def prefix   = task.ext.prefix ?: "${meta.id}"
+    """
+    run_resfinder.py \\
```

Comment:
option to specify CPU cores or threads for parallel computing?

Reply:
I didnt see any option to specify the CPUS - https://bitbucket.org/genomicepidemiology/resfinder/src/master/

Reply:
I guess the tool doesn't support it, closing!